public/docs/_examples/structural-directives/ts/src/app/scrap.txt

```diff
@@ -1,21 +0,0 @@
-// interesting but unused code

```

Comment:
Done. (I thought that this file showed up in the plnkr and zip, but I just checked and it doesn't.)